### ISSUE TITLE
Where did you learn to code? 

### ISSUE THREAD

**Issue by GoOpaque** [KingPixil/ama#9]

**Comment by kbrsh** [comment]
There are many resources I used to learn coding. One really good one is [FreeCodeCamp](https://freecodecamp.com). They teach you a lot of the needed skills for coding, including some challenges that are pretty tricky to get through. You build projects etc. But the best way I would recommend to learn is to imagine something you want to build, and begin. If you don't know where to start, google it. If you get stuck, google it. You will work towards something you want to build, and have a finished product in the end. 

TL;DR: FreeCodeCamp is a nice place to learn, and the best way (my opinion) is to build things you want, and make it. Learn by doing. 👍 

Good luck, and thanks for asking @GoOpaque 😄
